jsapp/xlform/src/view.row.coffee

```diff
@@ -22,9 +22,11 @@ getGroupFeatures = require('js/components/locking/lockingUtils').getGroupFeature
 LOCKING_RESTRICTIONS = require('js/components/locking/lockingConstants').LOCKING_RESTRICTIONS
 LOCKING_UI_CLASSNAMES = require('js/components/locking/lockingConstants').LOCKING_UI_CLASSNAMES
 $icons = require('./view.icons')
+# TODO: figure out how to switch to toast
 multiConfirm = require('js/alertify').multiConfirm
```

Comment:
Please create a new GH issue about switching this piece of code to toast. And add a link to it in comment here.

Reply:
Changed to 
```coffee
# TODO: port this and others from alertify.dialog to new modal system
# https://github.com/kobotoolbox/kpi/issues/3977
```